.NET/Samples/BotBuilder/Dialogs/RootDialog.cs

```diff
@@ -17,6 +17,13 @@ public async Task StartAsync(IDialogContext context)
             context.Wait(this.MessageReceivedAsync);
         }
 
+        private static string GetCurrentCultureCode()
+        {
+            // Use English as default culture since the this sample bot that does not include any localization resources
```

Comment:
I think there is an extra `the` in the line `[...] since the this sample bot [...]`.

Reply:
@Bill7zz this is still here!